service_client.go

```diff
@@ -114,6 +114,8 @@ func (client *ServiceClient) setMicroversionHeader(opts *RequestOpts) {
 		opts.MoreHeaders["X-OpenStack-Nova-API-Version"] = client.Microversion
 	case "sharev2":
 		opts.MoreHeaders["X-OpenStack-Manila-API-Version"] = client.Microversion
+	case "volume":
+		opts.MoreHeaders["X-OpenStack-Volume-API-Version"] = client.Microversion
```

Comment:
If `client.Microversion` is an empty string, this piece of code will never be reached. See here: https://github.com/j-griffith/gophercloud/blob/2af0cfeba7c8f0e894f945fb89ab8b0b0f7096a6/service_client.go#L61-L63